app/smc/pytest/recovery.py

```diff
@@ -0,0 +1,144 @@
+#!/bin/env python3
+
+# Copyright (c) 2025 Tenstorrent AI ULC
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import time
+import sys
+
+import pyluwen
+
+from twister_harness import DeviceAdapter
+
+# Import tt_boot_fs utilities
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "..", "scripts"))
```

Comment:
Would be kind of preferable to use a `Path` object (requires `from pathlib import Path` too)

```suggestion
sys.path.insert(0, Path(__file__).parents[3] / "scripts")
```